docs/libcurl/symbols-in-versions

```diff
@@ -126,6 +126,7 @@ CURLE_SSL_CACERT                7.10          7.62.0
 CURLE_SSL_CACERT_BADFILE        7.16.0
 CURLE_SSL_CERTPROBLEM           7.10
 CURLE_SSL_CIPHER                7.10
+CURLE_SSL_CLIENTCERT            7.76
```

Comment:
`7.76` is not a legitimate curl version and this is too late for `7.76.0` (we're in feature-freeze) so likely this can be subject for inclusion in `7.77.0`

Reply:
Changed to 7.77.0.

Reply:
Added documentation updated at docs/libcurl/libcurl-errors.3.  Our code uses secure transport on OSX and curl wrapper depends on the code to request/communicate/choose  a certificates from multiple certificates.  OpenSSL is used in our product on Linux platforms, so there might be a plan extend it at OpenSSL in future.